config/install/ivw_integration.settings.yml

```diff
@@ -1,27 +1,27 @@
 site: ''
 mobile_site: ''
-frabo: 'in'
-frabo_overridable: 0
-frabo_mobile: 'mo'
-frabo_mobile_overridable: 0
 code_template: '[ivw:offering]L[ivw:language]F[ivw:format]S[ivw:creator]H[ivw:homepage]D[ivw:delivery]A[ivw:app]P[ivw:paid]C[ivw:content]'
 responsive: 1
 mobile_width: 480
 offering_default: '01'
-offering_overrideable: 0
+offering_overridable: false
```

Comment:
The spelling error should also be corrected in IvwLookupServiceTest.php method setUpConfigFactoryMock() and in SettingsForm.php (several titles are wrong)